cmd/codeqlExecuteScan.go

```diff
@@ -394,6 +402,21 @@ func uploadProjectToGitHub(config *codeqlExecuteScanOptions, repoInfo *codeql.Re
 	return nil
 }
 
+func runCustomCommand(utils codeqlExecuteScanUtils, command string) error {
+	log.Entry().Infof("custom command will be run: %s", command)
+	cmd, err := shlex.Split(command)
+	if err != nil {
+		log.Entry().WithError(err).Errorf("failed to parse custom command %s", command)
+	}
+	err = utils.RunExecutable(cmd[0], cmd[1:]...)
```

Comment:
can we log the output of Split() here? that could help diagnose parsing/quoting issues later.